### PR TITLE
Ttbarhhexample

### PR DESCRIPTION
**Pull request by clementhelsens**
Clean ttbarhh example.
@cbernet, In order not to store isolation variables for specific particles ID, I added in the booklepton function in the ntuple.py a flag pflow that is True by default. I then turn it off for my analysis.

### CODE REVIEW

File: analyzers/Btagging.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from heppy.framework.analyzer import Analyzer
+
+class Btagging(Analyzer):
+    
+    def process(self, event):
+        jets = getattr(event, self.cfg_ana.input_objects)
+        bjets = [jet for jet in jets if self.cfg_ana.filter_func(jet)]
```
Comment (cbernet) [inline analyzers/Btagging.py]: Salut Clement!
no need for two loops, nor to run the filtering twice
do something like

    for jet in jets: 
         if filter_func(jet):
              bjets.append(jet)    
              jet.tag['b'] = True
         else:
              jet.tag['b'] = False

File: analyzers/M3Builder.py [hunk 2]
```diff
@@ -4,21 +4,23 @@
 import pprint 
 import itertools
 
-
 class M3Builder(Analyzer):
     
     def process(self, event):
         jets = getattr(event, self.cfg_ana.jets)
-        jets = [jet for jet in jets if self.cfg_ana.filter_func(jet)]
+        #jets = [jet for jet in jets if self.cfg_ana.filter_func(jet)]
```
Comment (cbernet) [inline analyzers/M3Builder.py]: remove

File: analyzers/ntuple.py [hunk 3]
```diff
@@ -96,10 +96,11 @@ def fillIso(tree, pName, iso):
     fill(tree, '{pName}_pt'.format(pName=pName), iso.sumpt )
     fill(tree, '{pName}_num'.format(pName=pName), iso.num )    
 
-def bookLepton( tree, pName ):
+def bookLepton( tree, pName, pflow=True ):
     bookParticle(tree, pName )
-    for pdgid in iso_pdgids:
-        bookIso(tree, '{pName}_iso{pdgid:d}'.format(pName=pName, pdgid=pdgid))
+    if pflow:
```
Comment (cbernet) [inline analyzers/ntuple.py]: sounds good!

**Comment by cbernet** [comment]
Hi Clement, 

Apart from a few comments, I can merge as soon as you confirm that the getting started instructions work for the FCC-ee case, https://twiki.cern.ch/twiki/bin/view/FCC/FccSoftwareGettingStarted#Getting_started_with_papas_FCC_e

Cheers!

**Comment by clementhelsens** [comment]
retest this please

**Pull request merged**